pkg/ingester/transfer.go

```diff
@@ -149,24 +149,18 @@ func (i *Ingester) TransferChunks(stream client.Ingester_TransferChunksServer) e
 func (i *Ingester) checkFromIngesterIsInLeavingState(ctx context.Context, fromIngesterID string) error {
 	v, err := i.lifecycler.KVStore.Get(ctx, ring.ConsulKey)
 	if err != nil {
-		return errors.Wrap(err, "TransferChunks: get ring")
+		return errors.Wrap(err, "get ring")
 	}
 	if v == nil {
-		return fmt.Errorf("TransferChunks: ring not found when checking state of source ingester")
+		return fmt.Errorf("ring not found when checking state of source ingester")
 	}
 	r, ok := v.(*ring.Desc)
 	if !ok || r == nil {
-		return fmt.Errorf("TransferChunks: ring not found, got %T", v)
-	}
-
-	if r.Ingesters == nil || r.Ingesters[fromIngesterID].State != ring.LEAVING {
-		return fmt.Errorf("TransferChunks: source ingester is not in a LEAVING state, found state=%v", r.Ingesters[fromIngesterID].State)
+		return fmt.Errorf("ring not found, got %T", v)
 	}
 
 	if r.Ingesters == nil || r.Ingesters[fromIngesterID].State != ring.LEAVING {
-		err = fmt.Errorf("source ingester is not in a LEAVING state, found state=%v", r.Ingesters[fromIngesterID].State)
-		util.Logger.Log("msg", "TransferChunks error", "err", err)
```

Comment:
Note to myself: removing this log is OK, cause the error will be logged by `Ingester.TransferOut()`.